packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.js

```diff
@@ -0,0 +1,105 @@
+
+import { mergeClasses } from '../../classify';
+import Button from '../Button';
+import { Message } from '../Field';
+import { Title } from '../Head';
+import { fullPageLoadingIndicator } from '../LoadingIndicator';
+import EditModal from './editModal';
+import defaultClasses from './accountInformationPage.css';
+import AccountInformationPageOperations from './accountInformationPage.gql.js';
+
+const AccountInformationPage = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const talonProps = useAccountInformationPage({
+        ...AccountInformationPageOperations
+    });
+
+    const {
+        loadDataError,
+        formErrors,
+        handleSubmit,
+        initialValues,
+        isDisabled,
+        isSignedIn,
+        activeChangePassword,
+        handleActivePassword,
+        isUpdateMode,
+        handleCancelUpdate,
+        showUpdateMode
+    } = talonProps;
+
+    if (!isSignedIn) {
+        return <Redirect to="/" />;
+    }
+
+    const errorMessage = loadDataError ? (
+        <Message>
+            {'Something went wrong. Please refresh and try again.'}
+        </Message>
+    ) : null;
+
+    let pageContent = null;
+    if (!initialValues) {
+        return fullPageLoadingIndicator;
+    } else {
+        const { customer } = initialValues;
+
+        pageContent = (
+            <Fragment>
+                <div className={classes.cardArea}>
+                    <div className={classes.lineItems}>
+                        <span className={classes.lineItemLabel}>{'Name'}</span>
+                        <span className={classes.lineItemValue}>{`${
+                            customer.firstname
+                        } ${customer.lastname}`}</span>
+                        <span className={classes.lineItemLabel}>{'Email'}</span>
+                        <span className={classes.lineItemValue}>
+                            {customer.email}
+                        </span>
+                        <span className={classes.lineItemLabel}>
+                            {'Password'}
+                        </span>
+                        <span className={classes.lineItemValue}>
+                            {'***********'}
+                        </span>
+                        <span className={classes.lineItemLabel} />
+                        <span className={classes.lineItemButton}>
+                            <Button
+                                className={classes.editInformationButton}
+                                disabled={false}
+                                onClick={showUpdateMode}
+                                priority="normal"
+                            >
+                                {'Edit'}
+                            </Button>
+                        </span>
+                    </div>
+                </div>
+                <EditModal
+                    informationData={customer}
```

Comment:
This is just `initialValues`.

```suggestion
                    initialValues={customer}
```